tools/docgen/docgen/channels.py

```diff
@@ -54,111 +55,17 @@ def process(self, input_file: str) -> ():
 
 
 class FabricChannel(Channel):
-    def __init__(self, input_dir: str, output_dir: str, notebooks: List[dict]):
+    def __init__(self, input_dir: str, output_dir: str, notebooks: List[dict], **kwargs):
         self.input_dir = input_dir
         self.output_dir = output_dir
         self.notebooks = notebooks
+        self.output_structure = kwargs.get("output_structure", "hierarchy")
```

Comment:
does it need to be kwargs or can we add an output structure argument explicitly here?